tool/run/qemu.inc

```diff
@@ -21,7 +21,6 @@ proc qemu_args { } {
 # XXX should by removed in favor of [have_include "exec/qemu"]
 #
 proc is_qemu_available { } {
-	if {[have_spec linux]} { return false }
```

Comment:
For base-linux-*traditional* this line should be kept, i.e., base-linux-hw may fall through.

Reply:
How would I check for base-linux-hw here? It is currently selected by choosing KERNEL=linux and BOARD=pc.

Reply:
untested sketch: `[expr [have_spec linux] && [board] == "linux"]` ?

Reply:
I fixed that in https://github.com/genodelabs/genode/pull/2829/commits/9d9f7e365bd801a4c73c0b883f8c81c3d0bf2d29.